### DISCUSSION TITLE
Ts unacceptable

### DISCUSSION THREAD

**Discussion by killgapwalker** [strongswan/strongswan#519] [General]
I'm making traffic without encapsulation , what need to change?
 I'm tryied  this config
conn NordVPN
 keyexchange=ikev2 
 dpdaction=restart 
 dpddelay=300s 
 eap_identity= nord username
 leftauth=eap-mschapv2 leftsourceip=%config
 right=il40.nordvpn.com
rightauth=pubkey 
Ike=myconfig
rightsubnet=10.6.0.3
rightid=%any 
type=tunnel
auto=add

Ts unnaceptable why ?

**Comment by tobiasbrunner** [comment]
Only the server knows. But most likely it doesn't like the value you configured for `rightsubnet`.

**Reply by killgapwalker** [reply]
This is not a problem...
